### PR TITLE
promote image of toolchain-operator

### PR DESCRIPTION
**Pull request by dipak-pawar**
We are using `openshift-ci` for `toolchain-operator`. We'd like to publish image to quay with every merge to master.

### CODE REVIEW

**Comment by droslean** [comment]
@smarterclayton Can you check where they should promote their images? They should promote the images in their namespace, right? like `kie-cloud` team did.

**Comment by smarterclayton** [comment]
Yes, and like knative

On May 7, 2019, at 8:31 AM, Nikolaos Moraitis <notifications@github.com>
wrote:

@smarterclayton <https://github.com/smarterclayton> Can you check where
they should promote their images? They should promote the images in their
namespace, right? like kie-cloud team did.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/release/pull/3704#issuecomment-490059650>,
or mute
the thread

**Comment by dipak-pawar** [comment]
@stevekuznetsov @smarterclayton @droslean Can you look into this when you have time?

File: ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml [hunk 1]
```diff
@@ -14,7 +14,10 @@ resources:
     requests:
       memory: 250Mi
       cpu: 100m
-
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
```
Comment (smarterclayton) [inline ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml]: You need to promote to a different namespace.

Comment (dipak-pawar) [reply ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml]: @smarterclayton thanks for your comments. addressed it in 261ffa1

**Pull request merged**